mail_environment/README.rst

```diff
@@ -0,0 +1,113 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+mail environment
+================
+
+This module allows to configure the incoming and outgoing mail servers
+using the server_environment mechanism.
+
+Installation
+============
+
+To install this module, you need to have the server_environment module
+installed and properly configured.
+
+Configuration
+=============
+
+With this module installed, the incoming and outgoing mail servers are
+in the server_environment_files module that you should provide.
+
+In the configuration file for each environment, you may use the
+sections 'outgoing_mail' and 'incoming_mail' to configure the default
+values for SMTP servers and the IMAP/POP servers respectively.
+
+Then for each server, you can (re)define values with a section named
+"outgoing_mail.resource_name" or "incoming_mail.resource_name" where
+"resource_name" is the name of your server.
+
+Exemple of config file :
+
+[outgoing_mail]
+smtp_host = smtp.myserver.com
+smtp_port = 587
+smtp_user =
+smtp_pass =
+smtp_encryption = ssl
+
+[outgoing_mail.openerp_smtp_server1]
+smtp_user = openerp
+smtp_pass = openerp
+
+[incoming_mail.openerp_pop_mail1]
+server = mail.myserver.com
+port = 110
+type = pop
+is_ssl = 0
+attach = 0
+original = 0
+user = openerp@myserver.com
+password = openerp
+
+
+Usage
+=====
+
+Once configured, your the values for the mail servers in the Odoo
```

Comment:
Once configured, Odoo will read the mail servers values from the 
configuration file related to each environment defined in the main Odoo file.